cypress/e2e/google.cy.js

```diff
@@ -0,0 +1,7 @@
+describe("This is my first cypress test", () => {
+    it("should have a title", () => {
+      cy.visit("https://www.google.com/");
+      cy.title().should("eq", "Google");
+    });
+  });
+  
```

Comment:
EOF

Reply:
probably line 7 has whitespaces, remove them and you will fix the EOF issue

Reply:
Remove space you have in line 7
![image](https://user-images.githubusercontent.com/36433884/180516980-46e3ac69-5d2d-4f77-b014-4219684ee154.png)
![image](https://user-images.githubusercontent.com/36433884/180517003-ce5a8104-34d5-42b1-aea7-50ae5e0c1721.png)